packages/frontend-v2/utils/plan/updatePlanOnDragEnd.ts

```diff
@@ -29,82 +29,95 @@ import { isCourseInTerm } from "./isCourseInTerm";
 export const updatePlanOnDragEnd = (
   plan: PlanModel<string>,
   draggedCourse: Active,
-  draggedOverTerm: Over
+  draggedOverTerm: Over | null = null
 ): PlanModel<string> => {
   const updatedPlan = produce(plan, (draftPlan) => {
     // grab all the terms across the years, since it's easier to work with a flat list of terms
     const scheduleTerms = flattenScheduleToTerms<string>(draftPlan.schedule);
 
-    const newTerm = scheduleTerms.find(
-      (term) => term.id === draggedOverTerm.id
-    );
-
-    if (!newTerm) {
-      throw new Error("Term the course is dragged over isn't found");
-    }
+    // If it is not being dragged over a droppable, then we delete the course
+    if (draggedOverTerm == null) {
+      const newTerm = scheduleTerms.find(term => term.classes.some((course) => course.id === draggedCourse.id));
```

Comment:
this is actually the old term right? nit but could u change the name bc its a little confusing w all of the local vars